src/main/java/fish/payara/extras/diagnostics/asadmin/CollectAsadmin.java

```diff
@@ -70,6 +70,9 @@ public class CollectAsadmin extends BaseAsadmin {
     @Param(name = ParamConstants.DOMAIN_XML_PARAM, optional = true, defaultValue = "true")
     private boolean collectDomainXml;
 
+    @Param(name = ParamConstants.OBFUSCATE_DOMAIN_XML_PARAM, optional = true, defaultValue = "true")
+    private boolean obfuscateDomainXml;
```

Comment:
Might be better to name this something more generic.

Perhaps just `obfuscate`